.pylintrc

```diff
@@ -5,4 +5,9 @@ disable=F0401,  # import loading
         E1101,  # function members
         R0801,  # duplicate code
         R0903,  # public methods
-        R0915   # too many statements
+        R0915,  # too many statements
+        R0915,  # too many statements
```

Comment:
there is a duplicate entry here

Reply:
Oop, good catch! Ironic that this happened on the "too many statements" statement...